lib/Admin/UserProfile.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Provide a admin area view for the plugin
+ *
+ * This file is used to markup the admin-facing aspects of the plugin.
+ *
+ * @link       http://log.pt/
+ * @since      1.0.0
+ *
+ * @package    RestKeyAuth
+ * @subpackage RestKeyAuth/lib/Admin
+ */
+?>
+<table class="form-table">
+	<tbody>
+		<tr>
+			<th scope="row">
+				<?php _e( 'REST API Key', 'key-auth' ); ?>
+			</th>
+			<td>
+				<?php
+					if ( ! empty( $api_key ) ) {
+						printf(
+							'<input type="text" name="rest_api_key" id="rest_api_key" value="%s" disabled="disabled" class="regular-text">',
+							$api_key
```

Comment:
You probably do need to escape it. They `api_key` may or may not contain `&`, `"`, `<` or `>` characters (which spell out `" onload="pwnage()` ;)). I wouldn't trust anything even if it comes from our own database, so these need to be escaped before you print them to HTML.